src/http_rest_api_handler.cpp

```diff
@@ -1123,4 +1126,12 @@ Status HttpRestApiHandler::processConfigStatusRequest(std::string& response, Mod
     return StatusCode::OK;
 }
 
+std::string urlDecode(const std::string& encoded) {
+    int output_length;
+    const auto decoded_value = curl_easy_unescape(nullptr, encoded.c_str(), static_cast<int>(encoded.length()), &output_length);
```

Comment:
This function may fail: https://curl.se/libcurl/c/curl_easy_unescape.html
Please add check for that and abort in such case